_posts/2023-09-08-training-series.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: "Learning SQL with Trino from the experts"
+author: "Manfred Moser"
+excerpt_separator: <!--more-->
+image: 
+show_pagenav: false
+---
+
+Do you think SQL stands for Silly Quacking Leopards and write all your code in
```

Comment:
It's generally poor practice to start an article with something that a reader can respond "no" to. I know it's a joke (and a good one), but if I have even a little bit of SQL knowledge, I'm immediately going to assume that this content is below my level. I might move it a couple lines deeper in so it's not immediately going to steer readers away.

Reply:
Good point .. let me rejig .. also .. I can link to the book .. since the cover is a leopardfrog (need to confirm that).